models/vgg16_mnist_beginner.py

```diff
@@ -0,0 +1,119 @@
+import tensorflow as tf 
+from tensorflow.keras.models import Model
+from tensorflow.keras.losses import MSE
+from tensorflow.keras.optimizers import Adam
+import numpy as np
+
+class Vgg16(tf.keras.Model):
+	def __init__(self,output_nodes):
+		super(Vgg16, self).__init__()
+		# layers needed
+		self.conv1_1 = tf.keras.layers.Conv2D(
```

Comment:
Import all layers used in the model at the beggining of the code, and write only layer name (`Dense`, `Conv2D`, etc.) in the model.